index.js

```diff
@@ -90,6 +92,8 @@ function focusTrap(element, userOptions) {
   function deactivate(deactivateOptions) {
     if (!state.active) return;
 
+    deactivateDelay(activeFocusDelay);
```

Comment:
I think we could use `clearTimeout` directly here instead of a wrapper function.

Reply:
done)